pkg/config/config.go

```diff
@@ -141,6 +141,11 @@ type Config struct {
 			AwsProfile         string `mapstructure:"aws_profile"`
 			AwsAccessKeyID     string `mapstructure:"aws_access_key_id"`
 			AwsSecretAccessKey string `mapstructure:"aws_secret_access_key"`
+
+			// Interval to run health check for the DynamoDB instance
```

Comment:
Code doc should start with the name of the func/variable - `HealthCheckInterval` in this case.
No need to specify the default - it is done in the code later in this file.